class-components/src/scripts/components/BottomSection.tsx

```diff
@@ -0,0 +1,131 @@
+import { Component } from 'react';
```

Comment:
Let's re-do the folder & file structure, please.

In React, the component (.tsx), its styles (.css/.scss), its types (placed in a separate .ts file) are usually kept in one folder.

So, your folder structure should go like this:
src
--App.tsx (file)
--App.css (file)
--assets (folder)
--components (folder)

[INSIDE components folder]
----BottomSection (folder)
[INSIDE BottomSection folder]
------ BottomSection.tsx
------ BottomSection.css
------ BottomSection.type.ts (some other files, if they are needed - like consts or maybe helpers for this component only)

[INSIDE components folder ]
----Search(folder)
[INSIDE Search folder]
------ Search.tsx
------ Search.css
------ Search.type.ts 

and so on.
The top-level folder scripts is not necessary.
Please let me know if you have questions.